include/swift/DependencyScan/SerializedModuleDependencyCacheFormat.h

```diff
@@ -60,6 +60,8 @@ using IsStaticField = BCFixed<1>;
 using IsForceLoadField = BCFixed<1>;
 /// A bit taht indicates whether or not an import statement is optional
 using IsOptionalImport = BCFixed<1>;
+/// A bit taht indicates whether or not an import statement is @_exported
```

Comment:
There's a repeated typo on `taht` with the lines above.

Reply:
Fixed. Tahnks!